raincloudy/const.py

```diff
@@ -2,7 +2,7 @@
 """Constants used by RainCloudy."""
 
 API_URL = 'https://wifiaquatimer.com'
-DAJAXICE_ENDPOINT = API_URL + '/dajaxice/webserver.get_cu_and_fu_status/'
```

Comment:
This was throwing a 500. Since the official site didn't seem to hit it anymore I moved to what seems to be the new endpoint.